tests/plugins/test_group_manifests.py

```diff
@@ -148,11 +152,6 @@ def mock_url_responses(docker_registry, test_images, worker_digests, version='2'
                 body = json.dumps({'error': 'INVALID MANIFEST'})
             for image_tag in test_images:
                 url = '{0}/v2/{1}/manifests/{2}'.format(registry, repo, image_tag.split(':')[1])
-
-                def verify_put_body(req):
-                    assert req.body == body.encode('utf-8')
-                    return (status, req.headers, '')
```

Comment:
'body' is only known in the closure for this function, so you can't move it ouside of here...

Reply:
...oh, weird, it works.